### ISSUE TITLE
Define http call method as `GET` by default

### ISSUE THREAD

**Issue by JBBianchi** [serverlessworkflow/specification#980]
**What would you like to be added**:
For the [HTTP Call](https://github.com/serverlessworkflow/specification/blob/main/dsl-reference.md#http-call) method not to be required and be defined as `get` if not specified.

**Why is this needed**:
It's probably opinionated but for me, GET feels like the "default" behavior of HTTP (even if it's not per se, it's still what happen under the hood each time we type in an address in our browser).

Wdyt ?  @cdavernas @ricardozanini @fjtirado @matthias-pichler

**Comment by matthias-pichler** [comment]
No strong opinions on my side, but maybe slightly against... my reasoning:

GET seems like a good default in the browser, end-user client world but for server side code less so

**Comment by cdavernas** [comment]
I am neither for nor against, quite the contrary 🤣

**Comment by fjtirado** [comment]
I agree with @matthias-pichler. GET is a good default for browsers, but for workflows maybe not so good. I'm slightly biased because is most examples I wrote for Kogito to illustrate 0.8 functionality I used post. And thats probably why, while designing a custom extension, I decided to set method (POST, GET, DELETE, PUT, PATCH...) parameter as mandatory. 
On the other hand, I do not really see a problem in omitting the method parameter for GET. At the end, it is saving lines. And, with mandatory or optional approach, POST has to be specified anyway (I do not think POST should be the default in any case)
